packages/rs-dpp/src/identity/identity.rs

```diff
@@ -65,7 +65,6 @@ pub struct Identity {
     pub balance: u64,
     #[bincode(with_serde)]
     pub revision: Revision,
-    #[bincode(with_serde)]
```

Comment:
@QuantumExplorer please check this change.  
Because I added `#[serde(untagged)]` to the AssetLockProof, bincode deserialization started to fail. From what I read in bincode [migration quide](https://github.com/bincode-org/bincode/blob/trunk/docs/migration_guide.md), one should use `#[bincode(with_serde)]` in case target item does not implement `Encode/Decode` and `AssetLockProof` actually does, so I think it should not be a problem.